2022.2/staged/core/meta.yaml.jinja

```diff
@@ -4,6 +4,7 @@ package:
 
 requirements:
   build:
+    - scikit-bio 0.5.6
```

Comment:
This works, but the idea is to use the conda build config for things like this - one stop shopping for all version pins.

Reply:
So I guess if there is a reason that the cbc doesn't work, maybe we should put that on a TODO list.

Reply:
We didn't think any of the recipes use `{{scikit-bio}}` as a version pin. We may misunderstand how the config is templated though.

Reply:
Totally, you're right - but for the metapackage it might be nice to "consume" the same file somehow. This would be a good enhancement!

Reply:
Agreed!

Reply:
After thinking a bit more about this, I think I'm okay with adopting what you've done here as the preferred strategy. The cbc.yml is for building individual packages, and i think setting the skbio pin there would imply that we should rebuild all of the individual packages, which doesn't really make sense. By choosing a compatible (and slightly more restrictive) pin here, you got the best of both worlds. I think what I'll do is just add some better documentation to the meta.yaml.jinja file to explain this.